js/rpg_managers/ImageManager.js

```diff
@@ -77,42 +85,43 @@ ImageManager.loadBitmap = function(folder, filename, hue, smooth) {
 };
 
 ImageManager.loadEmptyBitmap = function() {
-    var empty = this.cache.getItem('empty');
-    if (!empty) {
+    var empty = this._imageCache.get('empty');
+    if(!empty){
         empty = new Bitmap();
-        this.cache.setItem('empty', empty);
+        this._imageCache.add('empty', empty);
+        this._imageCache.reserve('empty', empty, this._systemReservationId);
     }
+
     return empty;
 };
 
 ImageManager.loadNormalBitmap = function(path, hue) {
-    var key = path + ':' + hue;
-    var bitmap = this.cache.getItem(key);
+    hue = hue || 0;
```

Comment:
Well, it's very detailed, but the intention of original code is below.

- loadBitmap's `hue`: can be `undefined`
- loadNormalBitmap's `hue`: can't be `undefined`

Could you adapt? (including `reserveXxx` and `requestXxx`)